client/src/components/frontend/Project/Hero.js

```diff
@@ -14,17 +14,78 @@ export default class ProjectHero extends Component {
     project: PropTypes.object
   };
 
-  listMakers() {
+  haveAvatars(creators, variant) {
+    let result = true;
+
+    creators.forEach(creator => {
+      const hasAvatarVariant = get(
+        creator.attributes,
+        `avatarStyles.${variant}`
+      );
+      if (!hasAvatarVariant) {
+        result = false;
+      }
+    });
+
+    // If none of the creators had a null avatar
+    return result;
+  }
+
+  listCreators() {
     const creators = get(this.props.project, "relationships.creators");
-    if (creators && creators.length > 0) {
+    if (!creators || creators.length <= 0) return null;
+    if (creators.length <= 2 && this.haveAvatars(creators, "smallSquare")) {
       return (
-        <section className="project-makers">
-          {this.props.project.relationships.creators.map(creator => {
+        <div className="project-creator-avatars">
+          {creators.map(creator => {
             return <Maker.Avatar key={creator.id} maker={creator} />;
           })}
-        </section>
+        </div>
       );
     }
+    return (
+      <div className="project-creator-list">
+        <ul>
+          {creators.map(creator => {
+            return (
+              <li key={creator.id} maker={creator}>
```

Comment:
This will throw a warning in the console because you can't have a maker prop on the `li` component. Why is this here?